orga/app/controllers/authenticated/campaigns/campaign/activity.js

```diff
@@ -28,16 +28,9 @@ export default class ActivityController extends Controller {
   }
 
   @action
-  triggerFiltering(filters) {
+  triggerFiltering(fieldName, value) {
+    this[fieldName] = value;
```

Comment:
`Question` : pourquoi on ne gère pas le cas ' undefined ' comme dans le filtre de controller/authenticated/organization-participants/list ( dans le commit ' use searchInputFilter in participants list' )

Reply:
Ça revient dans le dernier commit je crois 😄